### PR TITLE
Enfore use of u0.... for / units

### PR DESCRIPTION
**Pull request by pelikhan**
/ fractions should use a u0. decimal representation so that we don't have to apply transformation after packing.
- [x] encoding of i0.16 looks broken. I send 0.8 and get back -0.2 in return. It seems to rescale things in -0.5, 0.5. is that what we want? @mmoskal 
- [x] there is an annoying overflow issue with signed encodings for i0.16, 0.5 -> -0xfffe - it overflows. So if you take the slider all the way to the right and drops to 50%

### CODE REVIEW

File: services/ledpixel.md [hunk 1]
```diff
@@ -67,12 +67,12 @@ number, and followed by 3 bytes of color.
 
 ## Registers
 
-    rw brightness = 15: u8 / @ intensity
+    rw brightness: u0.8 / @ intensity
```
Comment (mmoskal) [inline services/ledpixel.md:70]:
```suggestion
    rw brightness = 0.05: u0.8 / @ intensity
```

**Review by mmoskal — APPROVED** [review]

File: services/ledpixel.md (hunk 1 above)
Comment (pelikhan) [reply services/ledpixel.md:70]: = 0.05 does not parse

Comment (mmoskal) [reply services/ledpixel.md:70]: I see. My TODO keeps growing: 

## Spectool
* check if events are under events in spectool
* check if typical max etc are in range in spectool
* `u8 /` to `u0.8 /` in spectool (error)
* check if something follows undefined-sized type (eg: repeats: x: string) in spectool
* const register at 0x80 doesn't cause compile error
* disallow i0.X (only i1.X is ok)
* = 0.05 (led pixel bright) does not parse

I guess you did one though :)

Comment (pelikhan) [reply services/ledpixel.md:70]: if you have a issue todo i can add notes :)

**Pull request merged**